client/ayon_core/hosts/houdini/api/colorspace.py

```diff
@@ -59,7 +59,7 @@ def get_colorspace_data(self):
 def get_default_display_view_colorspace():
     """Returns the colorspace attribute of the default (display, view) pair.
 
-    It's used for 'ociocolorspace' parm in OpenGL Node."""
+    It's used for 'ociocolorspace' param in OpenGL Node."""
```

Comment:
`parm` is actually the valid name for this in Houdini. See https://www.sidefx.com/docs/houdini/hom/hou/Parm.html

Reply:
```suggestion
    It's used for 'ociocolorspace' parm in OpenGL Node."""
```